crates/core/src/model/scalar/datetime.rs

```diff
@@ -134,6 +134,10 @@ const DATE_TIME_FORMAT: &[time::format_description::FormatItem<'static>] = time:
     "[year]-[month]-[day] [hour]:[minute]:[second] [offset_hour sign:mandatory][offset_minute]"
 );
 
+const DATE_TIME_FORMAT_SUBSEC: &[time::format_description::FormatItem<'static>] = time::macros::format_description!(
+    "[year]-[month]-[day] [hour]:[minute]:[second].[subsecond] [offset_hour sign:mandatory][offset_minute]"
+);
+
 impl fmt::Display for DateTime {
```

Comment:
I feel like we should update `Display` as well.  Maybe we check if there are milliseconds and render them if present?

Reply:
You mean checking if subseconds is > 0, right?

Reply:
Implemented here: 8fcc8154f5a8aee09a51ba21436535bc684f8d94

Reply:
Thanks!